java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -419,6 +419,71 @@ CompletableFuture<Map<String, Double>> zrangeWithScores(
      */
     CompletableFuture<Map<String, Double>> zrangeWithScores(String key, ScoredRangeQuery rangeQuery);
 
+    /**
+     * Stores a specified range of elements from the sorted set at <code>source</code>, into a new
+     * sorted set at <code>destination</code>. If <code>destination</code> doesn't exist, a new sorted
+     * set is created; if it exists, it's overwritten.<br>
+     * <code>ZRANGESTORE</code> can perform different types of range queries: by index (rank), by the
+     * score, or by lexicographical order.<br>
+     *
+     * @see <a href="https://redis.io/commands/zrangestore/">redis.io</a> for more details.
+     * @param destination The key for the destination sorted set.
+     * @param source The key of the source sorted set.
+     * @param rangeQuery The range query object representing the type of range query to perform.<br>
+     *     <ul>
+     *       <li>For range queries by index (rank), use {@link RangeByIndex}.
+     *       <li>For range queries by lexicographical order, use {@link RangeByLex}.
+     *       <li>For range queries by score, use {@link RangeByScore}.
+     *     </ul>
+     *
+     * @param reverse If <code>true</code>, reverses the sorted set, with index <code>0</code> as the
+     *     element with the highest score.
+     * @return The number of elements in the resulting sorted set.
```

Comment:
Is it overwritten or extended? Please reflect this in example by adding a comment

Reply:
What is dont understand your question. Are you asking this?
If <code>destination</code> doesn't exist, a new sorted et is created; if it exists, it's overwritten.<br>?